src/App.js

```diff
@@ -18,26 +20,29 @@ function App() {
     fetchData();
   }, []);
 
-  // TODO -- add event for button click to handle calling fetchBusinesses with zip / search
-
+  const handleChange = async () => {
+    const searchData = await fetchBusinesses(zipCode, search);
+    setBusinesses(searchData);
+    // console.log('display search info');
+    // TODO -- add event for button click to handle calling fetchBusinesses with zip / search
+    setLoading(false);
```

Comment:
Consider adding in error handling here, as a search without a zip code doesn't change the data.